### DISCUSSION TITLE
Calculator Assignment Ideas

### DISCUSSION THREAD

**Discussion by Souky-11** [Souky-11/T2S2T1VIRGO03_NEW#1] [General]
Hi Althea

I have created a remote repository for us to collaborate for Assignment 2. Any ideas on how we should assign the group work? Do you have any preferences of what you might like to do?

**Comment by Souky-11** [comment]
I have just updated the Development Branch - ready for you to clone

**Reply by altheatafe** [reply]
Hi Souky,

Thank you for setting it up.

What do you think if we assign the group work like below:

Section B (setting up the local & remote repository, development branch) + Section C (Mortgage Calculator)
Section C (Currency Conversion) + Section D (merging into development and release branch)
--> Then work on Section G together?
